src/app/infokit/infokit.pipe.ts

```diff
@@ -0,0 +1,51 @@
+import { Pipe, PipeTransform} from '@angular/core';
+
+@Pipe ({
+  name: 'infokitPipe'
+})
+
+export class InfokitPipe implements PipeTransform {
+  transform(dict: any[], args: any= []) {
+    let activity: any[] = [];
+    for (let key in dict) {
+      if (dict.hasOwnProperty(key)) {
+        switch (key) {
+          case 'malaria_def' : {
```

Comment:
The thing this code is doing is trying to change the values from APIs to values which can be displayed, In database Malaria activity is mentioned as malaria_def, this code is transforming it to Malaria, so that it can be displayed as a heading. If more activities are added in Database, more such rules will have to be added in the switch statement. It is written in this way to make addition of new activities easier.